infrastructure/README.md

```
@@ -0,0 +1,210 @@
+         - Task definition revision: LATEST
+      - Networking:
+         - VPC: owasp-nest-staging-vpc
+         - Security group name: select all with `owasp-nest-staging-` prefix.
+            (*Note*: temporary step, will be further improved)
+   - Click "Create"
+   - The task is now running... Click on the task ID to view Logs, Status, etc.
+   - Follow the same steps for `owasp-nest-staging-load-data` and `owasp-nest-staging-index-data`.
+
+## Cleaning Up
+
+   - To delete the deployment use the following command:
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 3

(MD007, ul-indent)